devices.php

```diff
@@ -1263,6 +1263,10 @@ function SNMPTest(){
 		$('#pdutest').dialog({minWidth: 850, position: { my: "center", at: "top", of: window },closeOnEscape: true });
 	});
 
+	// If the current hypervisor value is not ProxMox, set the field types to hidden
+	// to prevent browser/password manager autofill
+	$('#APIUsername,#APIPassword').attr("type","hidden");

```

Comment:
This will set those fields to hidden on every page load no matter what.

Reply:
Why are you touching the username anyway?  the password is the only piece here of concern.

https://github.com/opendcim/openDCIM/blob/f28a4a295311e4b260f362a82039e567ad4a0ad0/devices.php#L2071

Reply:
> This will set those fields to hidden on every page load no matter what.

My mistake, I missed a couple lines when cherry-picking from our local patch.  Will update



> Why are you touching the username anyway? the password is the only piece here of concern.
> 
> https://github.com/opendcim/openDCIM/blob/f28a4a295311e4b260f362a82039e567ad4a0ad0/devices.php#L2071

Because the password managers are also filling this, I presume we wouldn't want the database littered with random usernames in this field?  If that should never have any side-effects, then sure we can limit the scope to password